tests/yam/validate/validate_connectivity.pm

```diff
@@ -0,0 +1,23 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate connectivity.
+
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+    script_output("ip address show");
+    my $connectivity = get_var('INST_COPY_NETWORK', '1') ? 'full' : 'none';
```

Comment:
still deceiving, you don't set this variables, it is internal and default, it would be better to check for the case that really test:
`my $connectivity = check_var('INST_COPY_NETWORK', '0') ? 'none' : 'full';